packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -553,6 +573,11 @@ export default decorate([
       inputFullIntervalId: generateId,
       inputTimeoutId: generateId,
 
+      offlineBackupEnabled: ({
+        isFull,
+        propSettings,
+        settings = propSettings,
+      }) => isFull && Boolean(settings.getIn(['', 'offlineBackup'])),
```

Comment:
Is it necessary to check `isFull`? What does it mean when `isFull` is `false` but offline backup is enabled anyway?

Reply:
To keep the user preference.